src/libZilliqa/Zilliqa.cpp

```diff
@@ -148,6 +148,41 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_STATE("[IDENT] " << string(key.second).substr(0, 8));
   }
 
+  // Logging useful information
```

Comment:
The below information should only be displayed for the seed node. Consider checking if it's seed node.